javascripts/Start.js

```diff
@@ -3,12 +3,43 @@
 let start = false;
 let end = false;
 let pause = false;
+let request = require('request');
+let abc = "";
+abc.NOME = "Gabriel";
+const xpto = {};
+
+for (let i = 0; i < xpto.length; i++) {
+  xpto[i] = i;
+  console.log(xpto);
+}
+
+const array = [1, 2, 3];
+const b = array[5];
+
+async function teste() {
+  const res = await Promise
+    .resolve(foo())
+    .catch(err => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/gabholanda/module01-game/pullRequest?prid=8950836)
```suggestion
    .catch((err) => {
```